contracts/staking/contracts/src/StakingProxy.sol

```diff
@@ -202,32 +182,11 @@ contract StakingProxy is
                     LibStakingRichErrors.InvalidParamValueErrorCode.InvalidMinimumPoolStake
             ));
         }
-
-        // ERC20Proxy and Vault contract addresses must always be initialized
-        if (address(wethAssetProxy) == NIL_ADDRESS) {
```

Comment:
We should still have this test somewhere. Perhaps in the constructor for the logic contract.

Reply:
I could live without these `NIL_ADDRESS` tests tbh, since a wrong address is just as bad and, honestly, probably more likely to happen.

Reply:
I'll just add some basic string reverts in the `MixinDeploymentConstants` constructor for now. We can always remove them if they add too much to the codesize.